apps/sgx/Makefile

```diff
@@ -53,59 +53,27 @@ enclave_ldflags :=\
 	-Wl,-pie,-eenclave_entry -Wl,--export-dynamic\
 	-Wl,--defsym,__ImageBase=0 -Wl,--gc-sections
 
-app_cflags := -I$(SGX_SDK)/include -Ilib
-
-app_ldflags := -L$(SGX_SDK)/lib64\
-	-l$(urts_library_name) -l$(uservice_library_name) -lpthread\
-
 .PHONY: clean all
 
-all: lib/test_addone.signed.so bin/test_addone
-
-# Build rule for all-in-one TVM package library
-lib/tvm_runtime_pack.o: tvm_runtime_pack.cc lib/test_addone_t.o
-	@mkdir -p $(@D)
-	$(CXX) -c $< -o $@ $(pkg_cflags) $(pkg_ldflags) $(enclave_cxxflags) -g
+all: lib/test_addone.signed.so
```

Comment:
is ```signed.so``` a conventional way of naming sgx enclave library?

Reply:
[That's the standard that Intel uses](https://github.com/intel/linux-sgx/search?utf8=✓&q=".signed.so"). The same convention has been adopted by [other large SGX projects](https://github.com/baidu/rust-sgx-sdk/search?utf8=✓&q=".signed.so").

Reply:
k